integration-tests/src/test/java/oracle/weblogic/kubernetes/ItServerStartPolicy.java

```diff
@@ -427,9 +429,20 @@ public void testDomainRestart() {
     result = executeLifecycleScript(STOP_DOMAIN_SCRIPT, DOMAIN, null);
     assertTrue(result.contains("No changes needed"), "stopDomain.sh shouldn't make changes");
```

Comment:
Use constant STOP_DOMAIN_SCRIPT for "stopDomain.sh"

Reply:
This is a string to be when the test  fails.